src/components/UI/molecules/CardDivider/CardDivider.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Divider from '<atoms>/Divider/Divider';
+import Text from '<atoms>/Text/Text';
+import spacing from '<variables>/spacing';
+import { fontWeights, fontSizes } from '<variables>/fonts';
+import { textColors, borderColors } from '<variables>/colorPalette';
+
+/**
+ * @description - CardDivider Component
+ *
+ * @prop {string} text - divider text
+ * @prop {string} cardDividerMargin - divider margin
+ * @prop {string} width - maxWidth
+ * @prop {string} height - height
+ * @prop {string} dividerColor - color
+ * @prop {string} margin - margin
+ * @prop {string} fontSize - font Size
+ * @prop {string} textAlign - text Align
+ * @prop {string} textColor - color
+ * @prop {string} textDisplay - display
+ * @prop {string} textTransform - text transform
+ * @prop {string} fontWeight - fontWeight
+ * @prop {string} textPadding - text padding
+ * @prop {string} content - content
+ * @prop {string} cardDividerPadding - card divider padding
+ * @prop {string} display - container display
+ *
+ * @return {component} CardDivider
+ */
+const CardDivider = ({
+  text,
+  cardDividerMargin,
+  width,
+  height,
+  margin,
+  dividerColor,
+  fontSize,
+  textAlign,
+  textColor,
+  textDisplay,
+  textTransform,
+  fontWeight,
+  textPadding,
+  content,
+  display,
+  cardDividerPadding,
+}) => {
+  const SingleDivider = () => (
+    <Divider
+      width={width}
+      height={height}
+      margin={margin}
+      dividerColor={dividerColor}
+    />
+  );
+  const DividerText = () => (
+    <Text
+      fontSize={fontSize}
+      textAlign={textAlign}
+      color={textColor}
+      display={textDisplay}
+      textTransform={textTransform}
+      padding={textPadding}
+      content={content}
+      fontWeight={fontWeight}
+    >
+      {text}
+    </Text>
+  );
+
+  return (
+    <FlexContainer
+      display={display}
+      margin={cardDividerMargin}
+      padding={cardDividerPadding}
+    >
+      <SingleDivider />
+      {text && (
```

Comment:
What is the difference between text and DividerText

Reply:
`text` is a prop that is passed into the CardDivider. It is rendered in the Text component of the CardDivider. `DividerText` is actually a function that renders the Text component. I've renamed it to renderText().